recipes/orc/all/conandata.yml

```diff
@@ -8,3 +11,9 @@ sources:
   "1.9.5":
     url: "https://archive.apache.org/dist/orc/orc-1.9.5/orc-1.9.5.tar.gz"
     sha256: "6900b4e8a2e4e49275f4067bd0f838ad68330204305fd3f13a5ec519e9d71547"
+patches:
+  "2.1.0":
+    - patch_file: "patches/2.1.0-0001-fix-cmake.patch"
+      patch_description: "fix cmake build specific to 2.1.0"
+      patch_type: "backport"
+      patch_source: "https://github.com/apache/orc/pull/2108"
```

Comment:
Changes that in the patch but not in the referenced PR will be fixed in this PR: https://github.com/apache/orc/pull/2121